### DISCUSSION TITLE
如何动态更新“资源管理器”标题

### DISCUSSION THREAD

**Discussion by MilkWangStudio** [opensumi/core#3101] [Q&A] [answered]
需要切换目录树的版本，想在切换时将标题动态修改为“资源管理器[xxxx]”之类的名称。xxxx是根据接口动态返回的，请问有什么参数可以控制吗？

<img width="383" alt="image" src="https://github.com/opensumi/core/assets/15866249/8d7ce1e0-1230-4e2e-9026-bc05a6eed44c">

**Comment by erha19** [comment, marked as answer]
@MilkWangStudio 你可以通过 MainLayoutService 获取到响应面板的 handler，其中有 `updateTitle` 方法可用于更新面板 Title。 

例如 `const handler = this.mainLayoutService.getTabbarHandler(EXPLORER_CONTAINER_ID);`
